lms/static/sass/_build-discussion.scss

```diff
@@ -0,0 +1,16 @@
+// From _build-lms-v1.scss
```

Comment:
This particular thread of comments continues [here](https://github.com/edx/edx-platform/pull/13483#pullrequestreview-1985827).